webpack.config.babel.js

```diff
@@ -1,14 +1,14 @@
 import { resolve } from 'path'
 import ExtractTextPlugin from 'extract-text-webpack-plugin'
+import HtmlWebpackPlugin from 'html-webpack-plugin'
 
-export default function (env = 'development') {
```

Comment:
This was a mistake. `env` should be an object like `{ development: true }`, not a string.